Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabUndoHelpers.h

```diff
@@ -58,6 +59,32 @@ namespace AzToolsFramework
                 const PrefabDomValue& entityDomAfterUpdatingEntity,
                 AZ::EntityId entityId,
                 UndoSystem::URSequencePoint* undoBatch);
+
+            //! Helper function for deleting entities to source template with undo-redo support.
+            //! @param entityAliasPathList The alias path list for entities that will be removed.
+            //! @param parentEntityList The parent entity list that will be updated.
+            //! @param focusedInstance The current focused prefab instance.
+            //! @param undoBatch The undo batch node to register the removal undo nodes to.
+            void DeleteEntities(
```

Comment:
Can you describe why we need a DeleteEntities function and also a RemoveEntities function? It's not obvious by the function names what the differences are. I suggest renaming to make the differences more apparent.

Reply:
I was planning to remove it but recent changes in CreatePrefabInMemory just started depending on it.

I put a note:
```
//! Note: It does not include updating relevant parent entities.
void RemoveEntities(
```
But yeah I agree I should put more info about the differences or maybe rename it to "RemoveEntityDoms".